### DISCUSSION TITLE
maybe it was worth rewriting flameshot to Rust?

### DISCUSSION THREAD

**Discussion by NeilRiver** [flameshot-org/flameshot#3777] [General]
c++ developers have long since died out, the last one was seen alive since the construction of the pyramids of Cheops and he had no children to pass on the knowledge of c++.

The language is getting older and more narrow, while Rust is gaining momentum.

**Comment by FelixJochems** [comment]
You can tone down the theatrics

Someone is, of course, free to rewrite everything to rust but at this point in time it seems unlikely.

**Comment by mmahmoudian** [comment]
As @FelixJochems correctly pointed out, and as a FLOSS software, we of course encourage people to roll up their sleeves and implement the software they are willing to have/see.

C++ is what it is, and this project along with millions of other projects use it. When the time comes and C++ is completely obsolete, I do hope your Rust/Go/Zig/Nim close exists so that we all can happily migrate to yours.
